### ISSUE TITLE
Unable to create a new wiki page

### ISSUE THREAD

**Issue by BrokenEagle** [r888888888/danbooru#3410]
It was recently noted by user Nitrogen that wiki pages are unable to be created.  I confirmed this issue for myself on multiple browsers and on both servers.  The following exception get raised when trying to navigate to "http://danbooru.donmai.us/wiki_pages/new":

```
NoMethodError exception raised

    undefined method `+' for nil:NilClass
    app/models/post.rb:1135:in `fast_count'
    app/helpers/wiki_pages_helper.rb:9:in `wiki_page_post_previews'
    app/views/wiki_pages/new.html.erb:18:in `_app_views_wiki_pages_new_html_erb___4212615448533333190_69947375022360'
    app/controllers/wiki_pages_controller.rb:9:in `new'
```

**Comment by evazion** [comment]
3c6a613 fixed the immediate bug, which was that `Post.fast_count(nil)` failed if you had the "Deleted post filter" or "Safe mode" options turned on.

649969156ec9d6204b76f068c3fe8e4228fe0287 fixed it to not show the count in the navbar when creating a new wiki page.

200071922761b5c0fbe0604b3447a3857f10f014 fixed it to not use `Post.fast_count` for calculating the tag count on wiki pages. This was wrong because `Post.fast_count` depends on the current user's settings. The count shown for a tag on its wiki page could be different from the count shown everywhere else on the site, depending on your account settings.

**Comment by evazion** [comment]
Fixed a bug with 6499691 that made the post count not be shown on https://danbooru.donmai.us/wiki_pages/show_or_new?title=medium_hair.
